### PR TITLE
Create scss starter framework

### PR DESCRIPTION
**Pull request by instn**

### CODE REVIEW

File: src/frontend/package.json [hunk 1]
```diff
@@ -26,7 +26,9 @@
     "url": "https://github.com/openfun/instart-learning/issues"
   },
   "homepage": "https://github.com/openfun/instart-learning#readme",
-  "dependencies": {},
+  "dependencies": {
+    "codyhouse-framework": "^2.1.0"
```
Comment (jmaupetit) [inline src/frontend/package.json]: This should be in `devDependencies` as it's not required to run the project but rather as a dependency for the final built.

Comment (deodat) [reply src/frontend/package.json]: Right, I'm going to do a `yarn remove codyhouse-framework` and a `yarn add -D codyhouse-framework`, correct me if I'm wrong :)

**Comment by deodat** [comment]
The reference is now the branch create-scss-starter-framework-squash :
[https://github.com/cea-instn/instart-learning/pull/10](https://github.com/cea-instn/instart-learning/pull/10)

**Comment by deodat** [comment]
The reference is now the branch create-scss-starter-framework-squash :
[https://github.com/cea-instn/instart-learning/pull/10](https://github.com/cea-instn/instart-learning/pull/10)

**Pull request closed without merging**